.gitignore

```diff
@@ -1,3 +1,4 @@
+vendor
```

Comment:
Is there some reason that this needs to be `gitignore`'d? I don't think anything in the repo generates a vendor dir, so I don't think we ever expect it to exist.

Reply:
You have to `go mod vendor` to build the manager binary, and you either have to delete it every time you're about to push to your PR or never do `git add .` :) I don't think it hurts to keep it here

Reply:
> You have to `go mod vendor` to build the manager binary

I don't have to do that...